src/features/reviews/views/ReviewView.vue

```diff
@@ -205,7 +208,24 @@ const columns = computed(() => [
               deleteReview(row.original.id);
             },
           })
-        : "",
+        : h(
+            "div",
+            {
+              class:
+                "opacity-0 group-hover:opacity-100 transition-opacity duration-100",
+            },
+            [
+              h(mdicon, {
+                name: "share",
+                class: "cursor-pointer hover:text-primary transition-colors",
+                onClick: () => {
+                  const url = `${window.location.origin}/share/club/${clubId}/review/${row.original.id}`;
+                  void navigator.clipboard.writeText(url);
```

Comment:
I'd like to avoid the "void" promise assertion. In this case I would add a .then and a .catch statement and only show the toast success in the then, and a toast error in the error case